ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -82,6 +82,30 @@ msgstr ""
 msgid "Help"
 msgstr "Aider"
 
+#: ckanext/ontario_theme/templates/internal/organization/index.html
+msgid "Ministries"
+msgstr "Ministères"
+
+#: OVERRIDE: ckan/templates/snippets/search_result_text.html:27
+msgid "{number} ministry found for \"{query}\""
+msgid_plural "{number} ministries found for \"{query}\""
+msgstr[0] "{number} ministère trouvé pour \"{query}\""
```

Comment:
French translations should be using guillemets instead of quotation marks
```suggestion
msgstr[0] "{number} ministère trouvé pour « {query} »"
```

Reply:
Good catch! the quotes have been replaced by guimets now for Datasets, Ministries, and Groups searches.

Reply:
The guillemets should have non-breaking spaces as well, sorry!

Reply:
span does not seem to work here...

Reply:
This should suffice for now, we can make a ticket for it